nodes/core/logic/18-sort.html

```diff
@@ -0,0 +1,112 @@
+  limitations under the License.
+-->
+
+<!DOCTYPE html>
+
+<script type="text/x-red" data-template-name="sort">
+
+    <div class="form-row">
+        <label><i class="fa fa-dot-circle-o"></i> <span data-i18n="sort.key-type"></span></label>
+        <select id="node-input-keyType" style="width:200px;">
+            <option value="payload" data-i18n="sort.payload"></option>
+            <option value="exp" data-i18n="sort.exp"></option>
+        </select>
+    </div>
+
+    <div class="node-row-sort-key">
+        <div class="form-row">
+            <label><i class="fa fa-filter"></i> <span data-i18n="sort.key-exp"></span></label>
+            <input type="text" id="node-input-key" style="width:70%;">
+        </div>
+    </div>
+    
+    <div class="form-row">
+        <label><i class="fa fa-random"></i>  <span data-i18n="sort.order"></span></label>
+        <select id="node-input-order" style="width:200px;">
+            <option value="ascending" data-i18n="sort.ascending"></option>
+            <option value="descending" data-i18n="sort.descending"></option>
+        </select>
+    </div>
+
+    <div class="form-row" id="node-as_num">
+        <label>&nbsp;</label>
+        <input type="checkbox" id="node-input-as_num" style="display: inline-block; width: auto; vertical-align: top;">
+        <label for="node-input-as_num" style="width: 70%;" data-i18n="sort.as-number"></label>
+    </div>
+
+    <div class="form-row">
+        <label for="node-input-name"><i class="fa fa-tag"></i> <span data-i18n="node-red:common.label.name"></span></label>
+        <input type="text" id="node-input-name" data-i18n="[placeholder]node-red:common.label.name">
+    </div>
+</script>
+
+<script type="text/x-red" data-help-name="sort">
+    <p>A function that sorts a sequence of messages or payload of array type.</p>
+    <p>When paired with the <b>split</b> node, it will reorder the
+    messages.</p>
+    <p>The sorting order can be:</p>
+    <ul>
+        <li><b>ascending</b>,</li>
+        <li><b>descending</b>.</li>
+    </ul>
+    <p>For numbers, numerical ordering can be specified by a checkbox.</p>
+    <p>Sort key can be <code>payload</code> or any JSONata expression for sorting messages, element value or any JSONata expression for sorting payload of array type.</p>
+    <p>The sort node relies on the received messages to have <code>msg.parts</code> set for sorting messages. The split node generates this property, but can be manually created. It has the following properties:</p>
+    <p>
+        <ul>
+            <li><code>id</code> - an identifier for the group of messages</li>
+            <li><code>index</code> - the position within the group</li>
+            <li><code>count</code> - the total number of messages in the group</li>
+        </ul>
+    </p>
+    <p><b>Note:</b> This node internally keeps messages for its operation.  In order to prevent unexpected memory usage, maximum number of messages kept can be specified.
```

Comment:
What is the default  ?   maybe add      (Default is xxx messages)

Reply:
I will add default description in info message (no limit on number of messages).
Also found a missing fix of property name in settings.js.